### PR TITLE
Feature: Add manual donations recommendation

### PR DESCRIPTION
**Pull request by JoshuaHungDinh**
## Description

This pull request introduces a new feature to the Donations table - a "Manual Donations" button/link. When users hover over this element, a pop-up notice appears, providing information about the benefits of the manual donations add-on for those who may not be familiar with it. Clicking on the button/link opens a new window that follows a UTM link specifically designed for the manual donations add-on. Importantly, this button/link is only visible to users who do not have the manual donations feature installed and active.

After speaking with Jeffrey several other minor styling suggestions were included and applied to the other recommendations. These changes include font-size on the"Pro tip" messages, text-alignment on the "recurring recommendation button" while in mobile/menu view, as well as a contextual change from "Pro tip:" -> "PRO TIP:"

## Affects

Donation Table

## Visuals

https://github.com/impress-org/givewp/assets/75056371/00f32e34-b87a-410b-8146-cdd703fd9f69

## Testing Instructions

- Visit the Donations table
- Hover over the enter donations button and view the notice.
- Click the button, it should populate a new window with the appropriate UTM link.
- Activate Manual Donations add-on and revisit the Donations table. The link should be replaced with the regular "New Donation" - Manual donation button for creating manual donations.
 
## Pre-review Checklist

-   [ ] Acceptance criteria satisfied and marked in related issue
-   [ ] Relevant `@unreleased` tags included in DocBlocks
-   [ ] Includes unit tests
-   [ ] Reviewed by the designer (if follows a design)
-   [x] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204782112762770

### CODE REVIEW

**Review by JasonTheAdams — APPROVED** [review]
Looks good!

File: src/Donations/resources/components/DonationsListTable.tsx [hunk 1]
```diff
@@ -215,13 +214,32 @@ export default function DonationsListTable() {
...
+                >
+                    <span className={styles.manualDonationsAddOn}>{__('ADD-ON', 'give')}</span>
+                    {__('Enter Donations', 'give')}
+                    <span className={styles.manualDonationsMessage}>
+                        <img
+                            src={`${window.GiveDonations.pluginUrl}/assets/dist/images/admin/triangle-tip.svg`}
+                            alt={'manual donations'}
+                        />{' '}
+                        {__(
+                            'Keep track of your offline donations by creating donations directly from here, ensuring accurate records with minimal effort.',
```
Comment (Benunc) [inline src/Donations/resources/components/DonationsListTable.tsx:238]:
```suggestion
                            'Need to add in a record for a donation received elsewhere, or reconcile with the payment gateway? Add donation records with the Manual Donations add-on!',
```

**Review by Benunc — CHANGES_REQUESTED** [review]
Minor text change

**Comment by JoshuaHungDinh** [comment]
I promise no changes. Haha, thanks @Benunc!

**Review by Benunc — APPROVED** [review]
:shipit:

**Pull request merged**